modules/extensions/src/backend/index.ts

```diff
@@ -3,6 +3,11 @@ import * as sdk from 'botpress/sdk'
 const onServerStarted = async (bp: typeof sdk) => {}
 const onServerReady = async (bp: typeof sdk) => {
   const router = bp.http.createRouterForBot('extensions')
+  const config = (await bp.config.getBotpressConfig()).eventCollector
+
+  router.get('/events/update-frequency', async (_req, res) => {
+    res.send(config.collectionInterval)
```

Comment:
When sending data, a good habit to take is to always send an object. It's easier to know what it means, plus it allows extending it easily without breaking: res.send({ collectionInterval: config... })